game_patch/os/commands.cpp

```diff
@@ -104,6 +107,62 @@ FunHook<void()> verify_level_cmd_hook{
     }
 };
 
+bool check_can_use_cheat_cmd()
+{
+    return (rf::level.flags & rf::LEVEL_LOADED) && !rf::is_multi;
+}
+
+void handle_camera_command(FunHook<void()>& hook)
+{
+    if (!check_can_use_cheat_cmd()) {
+        if (!(rf::level.flags & rf::LEVEL_LOADED)) {
+            rf::console::print("No level loaded!");
+        }
+        else if (rf::is_multi) {
+            rf::console::print("That command can't be used in multiplayer.");
+        }
+        return;
+    }
```

Comment:
There is no point to calling `check_can_use_cheat_cmd`. Further it duplicates code from `heehoo_cmd_hook`. But it seems more trouble than worth it to fix.

Reply:
> There is no point to calling `check_can_use_cheat_cmd`.

The return in that if statement is what ensures the hud_msg or console::print text is not displayed if the command has no effect, though I guess I could just put a return in each of the two tests (level loaded and multiplayer) instead.

> Further it duplicates code from `heehoo_cmd_hook`. But it seems more trouble than worth it to fix.

This is the exact reason I had originally put the console::print in the bool. As is though yes, I think it's not worth the trouble to fix it. If it were replicating more than 2 very straightforward console messages I'd be more concerned.